### PR TITLE
[new] add Hand.js 1.3.11

### PR DESCRIPTION
**Pull request by F2X**
Hand.js integration to cdnjs (https://github.com/deltakosh/handjs)

### CODE REVIEW

File: ajax/libs/handjs/package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  ],
+  "license": "Apache-2.0",
+  "homepage": "http://github.com/deltakosh/handjs",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/deltakosh/handjs.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/deltakosht/handjs.git",
```
Comment (PeterDaveHello) [inline ajax/libs/handjs/package.json]: typo ...
`deltakosh` / `deltakosht`

File: ajax/libs/handjs/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    "basePath": "",
+    "files": [
+      "hand.js",
+      "hand.min.js"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/handjs/package.json]: Shouldn't also include css and hand.base?

Comment (F2X) [inline ajax/libs/handjs/package.json]: I don't think so, if I don't make a mistake and regarding to the gulpfile, css and hand.base are source files used to build "hand.js" and "hand.min.js". I will talk to the author to know if he can refactor his repo to split src and dist files, if you want.

Comment (PeterDaveHello) [inline ajax/libs/handjs/package.json]: Should confirm that if we'll need it.

Comment (F2X) [inline ajax/libs/handjs/package.json]: Ok

File: ajax/libs/handjs/package.json (hunk 1 above)
Comment (F2X) [inline ajax/libs/handjs/package.json]: I've fixed it

File: ajax/libs/handjs/package.json (hunk 2 above)
Comment (F2X) [inline ajax/libs/handjs/package.json]: It's ok right now. We've changed that on source repo

**Comment by PeterDaveHello** [comment]
Are we cool with the other files I said?

**Comment by F2X** [comment]
Yes, we are

**Comment by PeterDaveHello** [comment]
I didn't see any clarification about those files, did I miss something?

**Comment by F2X** [comment]
They are considered as source files, so there is no reason to publish them. It was seen with the author.

**Comment by PeterDaveHello** [comment]
Okay, please squash these commits into only one commit and rebase with our latest master branch and then I'll merge it, thanks!

**Comment by F2X** [comment]
Thanks, it was done

**Pull request merged**